docs/install_guides/installing-openvino-raspbian.md

```diff
@@ -7,7 +7,7 @@
   * There is also an open-source version of OpenVINO™ that can be compiled for arch64 (see `build instructions <https://github.com/openvinotoolkit/openvino/wiki/BuildingForRaspbianStretchOS>`_).
 @endsphinxdirective
 
-## Development and Target Platforms
+## Development and Target Systems 
 
 @sphinxdirective
```

Comment:
let's write in OS section that they are arm

Reply:
updated. :)